threejs/three.d.ts

```diff
@@ -5791,6 +5792,8 @@ declare module THREE {
     export class WireframeHelper extends Line {
         constructor(object: Object3D, hex?: number);
 
+        object: Object3D;

```

Comment:
You are right, I messed that up with the constructor definition. Will update the pull request